applications/FluidDynamicsApplication/custom_elements/alternative_d_vms_dem_coupled.cpp

```diff
@@ -447,94 +813,178 @@ void AlternativeDVMSDEMCoupled<TElementData>::AddVelocitySystem(
-                    double DD = tau_two * fluid_fraction * fluid_fraction * rData.DN_DX(i,d) * rData.DN_DX(j,e);
+                    double AGBeta = tau_one(d,d) * fluid_fraction * viscosity * AGradN[i] * rData.DN_DX(j,d) * fluid_fraction_gradient[e];
+                    double ADBeta = 2.0 / 3.0 * tau_one(d,d) * fluid_fraction * viscosity * AGradN[i] * fluid_fraction_gradient[d] * rData.DN_DX(j,e);
+                    double DD = tau_two * std::pow(fluid_fraction,2) * rData.DN_DX(i,d) * rData.DN_DX(j,e);
                     double DU = tau_two * fluid_fraction * fluid_fraction_gradient[e] * rData.DN_DX(i,d) * rData.N[j];
-                    double DBeta = 2.0 / 3.0 * tau_one(d,d) * fluid_fraction * kin_viscosity * rData.N[i] / dt * fluid_fraction_gradient[d] * rData.DN_DX(j,e);
-                    double DBetaA = 2.0 /3.0 * kin_viscosity * fluid_fraction * tau_one(d,d) * AGradN[j] * fluid_fraction_gradient[e] * rData.DN_DX(i,d);
+                    double GD = tau_two * fluid_fraction * fluid_fraction_gradient[d] * rData.DN_DX(j,e) * rData.N[i];
                     double GU = tau_two * rData.N[j] * rData.N[i] * fluid_fraction_gradient[d] * fluid_fraction_gradient[e];
-                    double GD = tau_two * fluid_fraction * rData.N[i] * fluid_fraction_gradient[d] * rData.DN_DX(j,e);
-                    double GBeta = tau_one(d,d) * fluid_fraction * kin_viscosity * rData.N[i] / dt * (rData.DN_DX(j,d) * fluid_fraction_gradient[e]);
-                    double AGBeta = tau_one(d,d) * fluid_fraction * kin_viscosity * AGradN[i] * (rData.DN_DX(j,d) * fluid_fraction_gradient[e]);
-                    double GBetaA = tau_one(d,d) * fluid_fraction * kin_viscosity * AGradN[j] * rData.DN_DX(i,e) * fluid_fraction_gradient[d];
-                    double GBetaG = 0.0;
-                    double GBetaD = 0.0;
-                    double GBetaSigma = 0.0;
+                    double GBetaA = tau_one(d,d) * fluid_fraction * viscosity * AGradN[j] * fluid_fraction_gradient[d] * rData.DN_DX(i,e);
+                    double LL_diag_1 = 0.0;
+                    double LL_diag_2 = 0.0;
+                    double LL_2 = 0.0;
+                    double LL_3 = 0.0;
+                    double LL_4 = 0.0;
+                    double LC_1 = 0.0;
+                    double LC_2 = 0.0;
+                    double LGBeta_1 = 0.0;
+                    double LGBeta_2 = 0.0;
+                    double LGBeta_3 = 0.0;
+                    double LGBeta_4 = 0.0;
+                    double LGBeta_5 = 0.0;
+                    double LDBeta_1 = 0.0;
+                    double LDBeta_2 = 0.0;
+                    double CL_1 = 0.0;
+                    double CL_2 = 0.0;
+                    double CC = 0.0;
+                    double CGBeta_1 = 0.0;
+                    double CGBeta_2 = 0.0;
+                    double CDBeta = 0.0;
+                    double GBetaL_1 = 0.0;
```

Comment:
it's ok, but maybe using an array for each subgroup of variables i.e.  `LGBeta` would simplify the null assignation

Reply:
I followed the way Cotela implemented `qs_vms.cpp` and `d_vms.cpp`. It is easier to fix posible errors.